pkg/k8sutil/client_config.go

```diff
@@ -24,3 +34,12 @@ func GetK8sInClusterConfig() (*rest.Config, error) {
 	}
 	return restCfg, nil
 }
+
+// MustCreateK8sClient creates a k8s client or panics.
+func MustCreateK8sClient(config *rest.Config) kubernetes.Interface {
```

Comment:
The refactoring now isn't strictly required since it's not being used anywhere but the sensor K8S client, but we may also keep it.

Reply:
Let's keep it to keep this helper function easier to find.